token-metadata/program/src/instruction/utility.rs

```diff
@@ -180,10 +182,71 @@ pub fn utilize(
 #[repr(C)]
 #[cfg_attr(feature = "serde-feature", derive(Serialize, Deserialize))]
 #[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Debug, Clone)]
-pub enum UseAssetArgs {
-    V1 {
-        use_count: u64,
+pub enum UtilityArgs {
+    LockV1 {
```

Comment:
That is a good question – I think all "use" relating things go here, so that is the motivation for the rename to `Utility`. We haven't ported the "use" things yet.

Reply:
These have been renamed to `lock` and `unlock`. Both were "promoted" to top-level instructions.